record-and-playback/screenshare/scripts/process/screenshare.rb

```
@@ -164,17 +164,15 @@
 if ret != 0
   raise "Generating closed caption files failed"
 end
-captions = JSON.load(File.new("#{process_dir}/captions.json", 'r'))
+captions = JSON.load(File.read("#{process_dir}/captions.json"))
```

RuboCop (1.86.2)

[warning] Useless assignment to variable - `captions`.

(Lint/UselessAssignment)